src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs

```diff
@@ -57,6 +57,19 @@ public PowerFxConfig(CultureInfo cultureInfo = null)
         {
         }
 
+        internal PowerFxConfig(CultureInfo culture, SymbolTable symbols, Features features)

```

Comment:
>PowerFxConfig [](http://example.com/codeflow?start=17&length=13)

why a new ctor?

Reply:
Need it to copy an existing config & change its locale
Used in WithCulture below, itself used in RenameDriver.cs.
Notice that we haven't any constructor taking a SymbolTable

Reply:
SymbolTable has a mutable setter.